static/include/app/apis/generated/app.md

```diff
@@ -1879,7 +1946,7 @@ Delete a registry item.
 
 **Parameters:**
 
-- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item.
+- `item_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the registry item. This is the namespace and name of the item in the form `namespace:name`.
```

Comment:
can you change the `"item-id"` in the code sample to `"your-namespace:your-name"`, just to help